### DISCUSSION TITLE
v4.1.0 – Feature Release

### DISCUSSION THREAD

**Discussion by Moonbase59** [Moonbase59/autocue#32] [Announcements]
#### New features

- New `liq_cue_file` handling, allows to ignore overrides for `cue_file` data if true. This is mainly for fast-changing files like news or time, for which LS/AzuraCast might not yet have updated the metadata.
  - _not set_ — default behaviour (metadata can override `cue_file` results)
  - `false` — don’t autocue (still use metadata if present)
  - `true` — `cue_file` results override metadata (special use cases)

For a more thorough explanation, see the [FAQ](FAQ.md#what-exactly-is-liq_cue_file-for-and-should-i-use-it)

**Update recommended.**

<hr /><em>This discussion was created from the release <a href='https://github.com/Moonbase59/autocue/releases/tag/v4.1.0'>v4.1.0 – Feature Release</a>.</em>

**Comment by gAlleb** [comment]
Yoohoo! Thanks, @Moonbase59! :) 

<img width="250" alt="2024-06-16_21 52 22" src="https://github.com/Moonbase59/autocue/assets/144447266/b2517a64-d99e-47d4-8aef-0882dab7b267"> da best))

**Reply by Moonbase59** [reply]
These logos you create are getting better and better, maybe we should talk ;-)
